source/common/quic/envoy_quic_client_stream.cc

```diff
@@ -407,5 +414,11 @@ void EnvoyQuicClientStream::onStreamError(absl::optional<bool> should_close_conn
 
 bool EnvoyQuicClientStream::hasPendingData() { return BufferedDataBytes() > 0; }
 
+void EnvoyQuicClientStream::enableHttpDatagramSupport() {
+  http_datagram_handler_ = std::make_unique<HttpDatagramHandler>(this);
+  ASSERT(response_decoder_ != nullptr);
```

Comment:
I don't think this check is needed. ASSERT() only happens in debug builds and malloc failure are very uncommon and basically impossible to defend against.

Reply:
Done.